.github/workflows/release.yml

```diff
@@ -48,28 +52,20 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
       - name: ${{ env.POETRY_VERSION }}-python-${{ matrix.python }}
         uses: docker/build-push-action@v2
+        if: ${{ matrix.variant == "buster" }}
         with:
           push: true
           tags: ${{ env.GHCR_IMAGE_REPOSITORY }}:${{ env.POETRY_VERSION }}-python-${{ matrix.python }}
           build-args: |
             POETRY_VERSION=${{ env.POETRY_VERSION }}
             PYTHON_VERSION=${{ matrix.python }}
-            VARIANT=buster
-      - name: ${{ env.POETRY_VERSION }}-python-${{ matrix.python }}-alpine
+            VARIANT=${{ matrix.variant }}
+      - name: ${{ env.POETRY_VERSION }}-python-${{ matrix.python }}-${{ matrix.variant }}
         uses: docker/build-push-action@v2
         with:
           push: true
-          tags: ${{ env.GHCR_IMAGE_REPOSITORY }}:${{ env.POETRY_VERSION }}-python-${{ matrix.python }}-alpine
+          tags: ${{ env.GHCR_IMAGE_REPOSITORY }}:${{ env.POETRY_VERSION }}-python-${{ matrix.python }}-${{ matrix.variant }}
           build-args: |
             POETRY_VERSION=${{ env.POETRY_VERSION }}
             PYTHON_VERSION=${{ matrix.python }}
-            VARIANT=alpine
-      - name: ${{ env.POETRY_VERSION }}-python-${{ matrix.python }}-slim
-        uses: docker/build-push-action@v2
-        with:
-          push: true
-          tags: ${{ env.GHCR_IMAGE_REPOSITORY }}:${{ env.POETRY_VERSION }}-python-${{ matrix.python }}-slim
-          build-args: |
-            POETRY_VERSION=${{ env.POETRY_VERSION }}
-            PYTHON_VERSION=${{ matrix.python }}
-            VARIANT=slim
+            VARIANT=${{ matrix.variant }}
```

Comment:
After this step (full tag), I suggest building the shortcuts, as described above[^1], based on the following conditions:

- [ ] `matrix.is_default_variant`:
    
    ```
    ${{ env.GHCR_IMAGE_REPOSITORY }}:${{ env.POETRY_VERSION }}-python-${{ matrix.python }}
    ```
- [ ] `matrix.is_default_python_version`:
    
    ```
    ${{ env.GHCR_IMAGE_REPOSITORY }}:${{ env.POETRY_VERSION }}-${{ matrix.variant }}
    ```
- [ ] `matrix.is_default_python_version && matrix.is_default_variant`:
    ```
    ${{ env.GHCR_IMAGE_REPOSITORY }}:${{ env.POETRY_VERSION }}
    ${{ env.GHCR_IMAGE_REPOSITORY }}:latest
    ```

[^1]: https://github.com/withlogicco/docker-poetry/pull/3/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R55

Reply:
Done 🎉